### DISCUSSION TITLE
你支持AI應用於天眼系統嗎？

### DISCUSSION THREAD

**Discussion by Doreen331** [1101-GS4719-C/Python-Community-GS4719-C#748] [General]
這學期的另一堂通識課在討論這個議題，通常一般人的答案都是不願意支持天眼系統，但其實天眼系統能帶給我們很多好處，例如降低犯罪率、和物聯網合作應用在交通領域，降低車禍事故等等，但需要犧牲一點民眾的隱私權，一開始我個人持反對意見，但換個立場想後，我有點改變我的想法，這是一個值得大家思考的問題，雖然可能永遠找不到正確答案和共識，但歡迎大家一起討論！附上一篇報導和一個呱吉和唐鳳在討論此議題的影片讓大家參考！
https://www.gvm.com.tw/article/60799
https://www.youtube.com/watch?v=41Px1sDO3rI

**Comment by HuangHsinTzu** [comment]
我認為天眼系統可以很大程度協助警察辦案，
甚至有威嚇大眾不要做壞事的效果!(讓大眾知道你如果做壞事警察很容易抓到你)
雖然有些人會有隱私疑慮
有些人會擔憂會有駭客趁機竊取個資，
但我認為現在網路上已經都找的到我們的資訊了
就算沒有天眼系統駭客一樣可以竊取個資
所以天眼系統其實沒有影響很大?
總而言之我支持天眼系統
(當然這只是我的個人看法而已)

**Comment by dinosaur1009** [comment]
我覺得天眼系統解決一些社會問題的確蠻有效率的，但也確實會有隱私問題......感覺可能不是能馬上實行的政策，但是個未來的趨勢！

**Comment by evanchou1204** [comment]
感覺最大的問題是要合意，很難在這個有爭議性的問題真的一個一個問，去取得全部人的同意。

**Comment by Pietra1226** [comment]
我覺得天眼系統牽涉到AI倫理的問題，該怎麼取捨便利跟隱私蠻兩難的。
因為每個人對於隱私的定義跟接受範圍不同，而且若是多數決也容易形成多數暴力。
公權力也有可能在暗中擴張，這是一個需要多面向衡量的問題。

**Comment by williamyen0623** [comment]
我犧牲一點隱私權也無仿，但一定要謹慎考慮，避免向科幻片那樣，被機器人支配（？

**Comment by Zoe06060606** [comment]
天眼技術能很有效的幫著警方解決問題 但法律和道得也是不可忽略的問題

**Comment by Evan0979** [comment]
超級可怕,感覺就會變成極權政府統治人民的武器!!

**Comment by StevenWanglolz** [comment]
我支持將AI應用於天眼，因為因為這樣可以造福全社會，前提是我們的資料受到非常嚴個的保護，而且只用於特定的用途!

**Comment by CPCpyt** [comment]
技術的運用感覺不是問題，
但我認為被濫用的可能性絕對不在少數，
畢竟絕對的權力帶來絕對的腐敗。

**Comment by swerfg03joe** [comment]
天眼系統可以降低犯罪率我持反對意見，應該是提高破案率才對，而這系統能不能用於解決交通問題，我只能說是本末倒置，多數交通問題都是道路規劃有問題使用路人產生疑惑、誤解，再來就是個人本身的教養問題，所以天眼說到底就是抓人很方便的工具，代價就是所有人的資料。

**Comment by Valentinaa0214** [comment]
我支持天眼系統，
天眼系統可以透過AI分析大量的影像和數據，
從而更快、更精確地發現和識別危險物質或威脅或監測和預測天氣狀況和環境變化。
以保證人們的安全或增加我們生活的便利性，
這些是我認為犧牲一點點隱私可以接受的，
但若是真正的應用了，也希望能夠制定一些準則以防止濫用。

**Comment by Sylvie0601** [comment]
我其實保持中立態度，因為如果推出天眼系統的話，真的對於社會來說有很大幫助，但我覺得在隱私問題上面沒有解決的話，我應該是不會到贊成的態度，感覺需要推出比較嚴謹的配套措施和政策，這樣我應該才會比較支持天眼系統。

**Comment by yuxinlin93** [comment]
在各地安裝天眼感覺可以給警方辦案有很大的幫助，對治安會有很大的提升，但這樣感覺真的很沒有隱私，而且如果被駭客入侵，造成的損失不知道會有多少。不是不支持，但覺得應該在真正上路前先訂定好無缺失的法規，還要先有小區域的測試，確定沒問題再上路會比較安全

**Comment by PaulSu0905** [comment]
我覺得這就要考驗法律制定的方面了。需要在保護公眾安全和個人隱私之間找到一個平衡點，明確的法律法規和監管機制，以確保天眼系統的使用是合法、透明和受到監督的。

**Comment by lyien1021** [comment]
我之前也有修過這門課，還記得當時我一開始是80%支持，因為我有聽犯罪故事podcast的習慣，我覺得那些受害者真的太可憐了，要是有天眼系統，很多寶貴的性命不會消失，不過經過還組員交流後，我發現我沒考慮到天眼系統背後的隱憂，像是被政府監督之類的，有可能造成政府權力過大的情況，因此我後來變成60%支持了（對我還是支持）

**Comment by larry40305** [comment]
我認為要有完整的法規來規範天眼系統，不然這項技術就會淪落成犯罪工具

**Comment by aa103967622** [comment]
我覺得天眼的系統能夠保持良好的社會治安，畢竟在城市的每個角落監視著，讓所有的犯罪無所遁形，也能對社會大眾起到警示的作用，降低犯罪率！但是，要是這樣的技術被利用在錯的地方，也相當恐怖，比如說駭客將此系統運用在個資買賣，這樣一來，所有人的隱私也將無法被保護，甚至可能反而讓人們陷入某種危機當中。

**Comment by youtinglin** [comment]
雖然運用天眼可以很好的掌控治安，我知道有的國家天眼系統非常的強大，可以辨識一個人在哪個位置，但也使得缺乏隱私權的保障，除非有完善的法律去維護我們個人的個資權益，但不得不說這系統的強大甚至可以使交通流量的控管更有效率。
